semantic_conventions/trace/database.yaml

```
@@ -251,6 +251,69 @@ groups:
           brief: >
             The name of the keyspace being accessed. To be used instead of the generic `db.name` attribute.
           examples: 'mykeyspace'
+        - id: page_size
+          type: number
+          tag: call-level-tech-specific
+          brief: >
+            The page size of the query.
+          examples: [5000]
+        - id: consistency_level
+          tag: call-level-tech-specific
+          brief: >
+            The consistency level of the query. Based on consistency values from [cql](https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/dml/dmlConfigConsistency.html).
```

```suggestion
            The consistency level of the query. Based on consistency values from [CQL](https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/dml/dmlConfigConsistency.html).
```
Or did you mean to write `cqlsh`?

---

CQL is fine. Either work. I'll use CQL.